server/server.go

```diff
@@ -132,12 +146,14 @@ func (c *Conn) write(v interface{}) error {
 
 // UpdateState sends registered users and open games to the client.
 func (c *Conn) UpdateState(id UserId) error {
-	if err := c.UserRegistered(users); err != nil {
-		log.Println(users, "UserRegistered")
+	state.Lock()
```

Comment:
Just by virtue of having a `Mutex` on `state`, we're able to call `state.Lock()`? That seems wild to me. (Like, we wouldn't call `state.Mutex.Lock()`? That's more what I'd expect.)